flight/Libraries/inc/morsel.h

```diff
@@ -0,0 +1,33 @@
+// Morse Library header
```

Comment:
Any reason for the C++ style comments around this header? Our code style guide doesn't allow them (although I know we all use them from time to time).